generic/tests/cfg/unattended_install.cfg

```diff
@@ -83,8 +83,22 @@
         - large_image:
             only unattended_install.cdrom
             only qcow2 qcow2v3
-            image_size_image1 = 2300G
-            image_size = 2300G
```

Comment:
what's the size for default, rhel8 test will failed with this cfg

Reply:
for Max boot disk size and max data disk size:
rhel 6.x guest for system disk ---------- 2TB
rhel 6.x guest for data disk \---------- 16TB
rhel 7.x guest for system disk ---------- 16TB
rhel 7.x guest for data disk \---------- 500TB
For windows:
guest for system disk ---------- 16TB
guest for data disk \---------- 16TB

During our testing, create bigger file than supported Max size, to make sure no crash issue.

Reply:
Agree with @suqinhuang should have the default values.

Reply:
@luckyh updated, many thanks